### DISCUSSION TITLE
SB5.1 - OCI - OpsAdministration/Core/GraylogMongoImport event_notifications error

### DISCUSSION THREAD

**Discussion by heungheung** [Sunbird-Ed/Community#356] [Installation] [answered]
Hit an error while running the 
  OpsAdminstration/Core/GraylogMongoImport 
job with Release 5.1.0. The error is w.r.t. event_notifications

```
failed: [10.0.2.10] (item=event_notifications) => {
    "ansible_loop_var": "item",
    "changed": false,
    "item": "event_notifications"
}

MSG:

AnsibleUndefinedVariable: 'alerts_mailing_list' is undefined
```

alerts_mailing_list is in the Core/common.yml but cannot populate to the task. (private repo ansible inventory)

**Comment by beepdot** [comment, marked as answer]
Is the host group present under `[core:children]`?
https://github.com/project-sunbird/sunbird-devops/blob/release-5.1.0/private_repo/ansible/inventory/dev/Core/hosts#L146

**Reply by heungheung** [reply]
Thanks!

Sorry that I've totally missed the commit 664f931df9f36d273ce854cf85deff9e0dbdbcc4 last week.

Once again, thanks a lot.
